test/xUnit/csharp/test_Runspace.cs

```diff
@@ -2,9 +2,9 @@
 // Licensed under the MIT License.
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-linux`
TestAppDomainProcessExitEvenHandlerNotLeaking
```powershell
Assert.Contains() Failure
Not found: (filter expression)
In value:  Delegate[] [EventHandler { Method = Void DisposeOnShutdown(System.Object, System.EventArgs), Target = null }]
   at PSTests.Sequential.RunspaceTests.TestAppDomainProcessExitEvenHandlerNotLeaking() in /home/vsts/work/1/s/test/xUnit/csharp/test_Runspace.cs:line 118
```